misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -112,5 +113,26 @@ abstract class PathAssertionsSpec(
             }
         }
     }
+
+    describeFun(endsNotWith.name) {
```

Comment:
personally I would fuse this describe with the above one as this is the opposite of endsWith. This way you could reuse the `context...` but it's also fine if you leave it this way

Reply:
I see that all others are also this way, so let's keep it for now